File: delete_data.go
// Random Go struct generated on 2025-06-03
package main

import (
    "encoding/json"
    "fmt"
    "time"
)

type DeleteUser struct {
    Name      string    `json:"name"`
    Value     int       `json:"value"`
    Active    bool      `json:"active"`
    CreatedAt time.Time `json:"created_at"`
}

func NewDeleteUser(name string, value int) *DeleteUser {
    return &DeleteUser{
        Name:      name,
        Value:     value,
        Active:    true,
        CreatedAt: time.Now(),
    }
}

func (s *DeleteUser) UpdateValue(newValue int) {
    s.Value = newValue
}

func (s *DeleteUser) ToJSON() ([]byte, error) {
    return json.Marshal(s)
}

func main() {
    obj := NewDeleteUser("sample", 70)
    data, _ := obj.ToJSON()
    fmt.Println(string(data))
}

File: delete_data.rb
# Random Ruby class generated on 2025-06-11

class SetUser
  attr_reader :name, :value, :created_at
  attr_writer :name, :value

  def initialize(name, value = 1)
    @name = name
    @value = value
    @created_at = Time.now
  end

  def to_hash
    {
      name: @name,
      value: @value,
      created_at: @created_at.iso8601
    }
  end

  def update_value(new_value)
    old_value = @value
    @value = new_value
    "Updated from #{old_value} to #{new_value}"
  end
end

# Example usage
obj = SetUser.new("sample", 28)
puts obj.to_hash
